sass/_variables.scss

```diff
@@ -0,0 +1,18 @@
+/*- fonts -*/
+$main-font: Helvetica, Arial, "Lucida Grande", sans-serif;
+
+/*- colors -*/
+
+$alpha: #FF6124;
+$beta: #585050;
+$gamma: #989595;
+$omega: #703A20;
+$kappa: #C1530D;
+
+$serv-cont-background: $alpha;
```

Comment:
Along those same lines, try to avoid unnecessary abbreviations in your variable names. I don't know what `serv-cont` is.